components/style/properties/longhand/inherited_text.mako.rs

```diff
@@ -866,27 +882,23 @@ ${helpers.single_keyword("text-align-last",
                     } else {
                         ShapeKeyword::Sesame
                     };
-                    computed_value::T::Keyword(computed_value::Keyword {
-                        fill: keyword.fill.unwrap_or(true),
-                        shape: keyword.shape.unwrap_or(default_shape)
+                    computed_value::T::Keyword(computed_value::KeywordValue {
+                        fill: keyword.fill().unwrap_or(true),
+                        shape: keyword.shape().unwrap_or(default_shape),
                     })
                 },
                 SpecifiedValue::None => computed_value::T::None,
                 SpecifiedValue::String(ref s) => {
-                    let string = if s.len() > 1 {
-                        s.chars().nth(0).unwrap().to_string()
-                    } else { s.clone() };
+                    let string = s.chars().next().unwrap_or_default().to_string();
```

Comment:
This is not the same of what I posted before.

This is using `char::default` which returns a non-empty string with a null (`\0`) char, which is wrong in this case (or at least not the intended thing because the resulting string won't be empty).